kolibri/plugins/facility/assets/src/views/DataPage/LearnMoreModal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+
+  <KModal
+    :title="logType === 'summary' ? $tr('summaryLogs') : $tr('sessionLogs')"
+    :submitText="$tr('close')"
+    size="medium"
+    @cancel="closeModal"
+    @submit="closeModal"
+  >
+    <p class="description">
+      {{ getMessage() }}
+    </p>
+  </KModal>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'LearnMoreModal',
+    props: {
+      logType: {
+        type: String,
+        required: true,
+      },
```

Comment:
Since your message parsing wouldn't fail but silently handle log types that aren't `summary` nor `session`, you could add a validator to ensure that this `logType` prop is one of those.